.travis/install.sh

```diff
@@ -44,19 +44,10 @@ else
   # Fallback
   TAG=$(git rev-parse --abbrev-ref HEAD | tr / _)
 fi
-
-if [ -e $TRAVIS_BUILD_DIR/../pulp_ansible ]; then
-  PULP_ANSIBLE=./pulp_ansible
-else
-  PULP_ANSIBLE=git+https://github.com/pulp/pulp_ansible.git@master
-fi
-
-mkdir vars
```

Comment:
Removing `pulp_ansible` from `additional_plugins` will remove this piece of code